src/js/themes/hpe.js

```diff
@@ -313,7 +313,7 @@ export const hpe = deepFreeze({
       background: { color: 'brand' },
       color: 'text-primary-button',
       font: { weight: 'bold' },
-      icon: Hpe,
+      icon: <Hpe />,
```

Comment:
Need to merge master? Seems like these revert what Eric just submitted?

Reply:
Reverts were intended due to errors when rendering icons. The fix to add `react` as an external in package.json wasn't reverted though, since that fix is necessary.